fab/environments.yml

```diff
@@ -143,7 +143,7 @@ production:
       email_queue:
         concurrency: 2
       repeat_record_queue:
-        concurrency: 6
+        concurrency: 20
```

Comment:
i think this is the right number to start off with, but if all goes well i think we can even make it larger